public/app/features/plugins/routes.ts

```diff
@@ -0,0 +1,59 @@
+import { SafeDynamicImport } from 'app/core/components/DynamicImports/SafeDynamicImport';
+import { config } from 'app/core/config';
+import { RouteDescriptor } from 'app/core/navigation/types';
+
+export function getPluginsAdminRoutes(cfg = config): RouteDescriptor[] {
+  if (!cfg.pluginAdminEnabled) {
+    return [
+      {
+        path: '/plugins',
+        component: SafeDynamicImport(() => import(/* webpackChunkName: "PluginListPage" */ './PluginListPage')),
+      },
+      {
+        path: '/plugins/browse',
```

Comment:
nit: I think path can accept an [array of routes](https://reactrouter.com/web/api/Route/path-string-string) which we could then use for these three `notEnabled` routes?

Reply:
that would have been nice but looks like you only can pass a string value.